src/SOS/extensions/extensions.cpp

```diff
@@ -25,6 +26,7 @@ Extensions* Extensions::s_extensions = nullptr;
 extern "C" HRESULT InitializeHostServices(
     IUnknown* punk)
 {
+    g_hostingInitialized = true;
```

Comment:
Even it fails this flag should be set.  It means that it was called from managed code and the hostcoreclr.cpp/hostdesktop.cpp doesn't need to be called. This is only necessary now because the TestDbgEng host called HostServices.Initialize() directly instead of it being called from hostcoreclr.cpp/hostdesktop.cpp.